virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -72,6 +72,24 @@ The SVVP Certification applies to:
 
 //CNV-28720 Release note: CHANGE
 
+//CNV-28724 Release note: storage profile defaults
+* `claimPropertySets` are now populated by default in storage profiles for the following additional provisioners:
```

Comment:
Suggestion: "The access mode and volume mode fields in storage profiles are populated automatically with their optimal values for the following Containerized Storage Interface (CSI) provisioners:"

I think this is a little clearer than `claimPropertySets`, which might be obscure for most of our users.

Reply:
Done